Mergin/utils.py

```diff
@@ -129,9 +133,56 @@ def get_qgis_version_str():
     return "{}.{}.{}".format(qgis_ver_major, qgis_ver_minor, qgis_ver_patch)
 
 
-def get_plugin_version():
+def plugin_version():
     with open(os.path.join(os.path.dirname(__file__), "metadata.txt"), 'r') as f:
         config = configparser.ConfigParser()
         config.read_file(f)
-        version = config["general"]["version"]
+    return config["general"]["version"]
+
+
+def get_plugin_version():
+    version = plugin_version()
     return "Plugin/" + version + " QGIS/" + get_qgis_version_str()
+
+
+def send_logs(username, logfile):
+    """ Send mergin-client logs to dedicated server
+
+    :param logfile: path to logfile
+    :returns: name of submitted file, error message
+    """
+    mergin_url, _, _ = get_mergin_auth()
+    system = platform.system()
```

Comment:
can we have system in lowercase, so that it's more consistent with input (also using lowercase for system)